Content.Tests/DMTests.cs

```diff
@@ -111,7 +111,7 @@ public void TestFiles(string sourceFile, DMTestFlags testFlags) {
 
         DreamThread.Run("RunTest", async (state) => {
             if (_objectTree.TryGetGlobalProc("RunTest", out DreamProc? proc)) {
-                callTask = state.Call(proc, null, null);
+                callTask = state.Call(proc, _dreamMan.WorldInstance, null);
```

Comment:
`/proc/RunTest()` is a global proc so giving it `world` as its `src` would be incorrect. The method instead needs this argument marked as nullable.